src/main/java/seedu/address/logic/parser/ArgumentMultimap.java

```diff
@@ -51,6 +51,13 @@ public List<String> getAllValues(Prefix prefix) {
         return new ArrayList<>(argMultimap.get(prefix));
     }
 
+    /**
+     * Returns if the ArgMultimap contains the given prefix or not.
+     */
+    public boolean hasGivenPrefix(Prefix prefix) {
+        return this.argMultimap.containsKey(prefix);
+    }
+
```

Comment:
I think the method name should be more concise such as `containsPrefix(Prefix)`

Reply:
Have changed accordingly